curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/66475bab40e5125c84b03337.md

```diff
@@ -0,0 +1,170 @@
+---
+id: 66475bab40e5125c84b03337
+title: Step 15
+challengeType: 20
+dashedName: step-15
+---
+
+# --description--
+
+Now that you have the coordinates like that, find the max value between all the x and the max value between all the y. Save the first in a new variable named `x_max` and the second in a variable named `y_max`.
```

Comment:
IMO it is better to specify why we need the max values here instead of in the next steps.